src/python/pants/rules/core/test.py

```diff
@@ -160,15 +167,22 @@ def register_options(cls, register) -> None:
             "--debug",
             type=bool,
             default=False,
-            help="Run a single test target in an interactive process. This is necessary, for example, when you add "
-            "breakpoints in your code.",
+            help="Run a single test target in an interactive process. This is necessary, for "
+            "example, when you add breakpoints in your code.",
         )
         register(
             "--run-coverage",
             type=bool,
             default=False,
             help="Generate a coverage report for this test run.",
         )
+        register(
+            "--open-coverage",
+            type=bool,
+            default=False,
+            help="If a coverage report file is generated, open it on the local system if the "
+            "system supports this.",
```

Comment:
I've seen you say this on reviews before and forgotten the advice. Although I like it, I will continue to forget. Since formatting is automatic now, I'll embrace that and let this stand as correct, since pretty / weird are no longer really valid words once you've picked an auto-formatter.